Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -384,23 +390,28 @@ public bool OpenSolution(string solutionFolder)
                 SolutionUpgrade.ClearPreviousScans();
                 if (SolutionUpgrade.IsGingerUpgradeNeeded(solutionFolder, solutionFiles))
                 {
+                    Console.WriteLine("Error: Current Ginger version");
                     Reporter.ToLog(eLogLevel.WARN, "Loading Solution- Error: Current Ginger version can't load the Solution because it includes items from higher Ginger version");
                     return false;
                 }
-
+                
                 Reporter.ToLog(eLogLevel.DEBUG, "Loading Solution- Loading Solution xml into object");
                 Solution solution = Solution.LoadSolution(solutionFile);
+                Console.WriteLine("Solution Laaded Successfully");
                 if (solution == null)
                 {
                     Reporter.ToUser(eUserMsgKey.SolutionLoadError, "Failed to load the Solution file");
                     Reporter.ToLog(eLogLevel.WARN, "Loading Solution- Error: Failed to load the Solution file");
+                    Console.WriteLine("Solution is null");
                     return false;
                 }
 
                 Reporter.ToLog(eLogLevel.DEBUG, "Loading Solution- Creating Items Repository");
                 SolutionRepository = GingerSolutionRepository.CreateGingerSolutionRepository();
                 SolutionRepository.Open(solutionFolder);
 
+                Console.WriteLine("Solution loaded in SR");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)